Flexbox2/style2.css

```
@@ -0,0 +1,62 @@
+body{
+    border: 0;
+    margin: 0;
+}
+
+.container{
+    height: 600px;
+    width: 600px;
+    display: flex;
+    gap: 16px;
+}
+
+.parent3{
+    display: flex;
+    flex-direction: row;
+    gap: 16px;
+}
+
+.parent2{
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+}
+
+.parent1{
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+}
+
+.box1{
+    background-color: turquoise;
+    height: 200px;
+    width: 200px;
+}
+
+.box2{
+    background-color: yellow;
+    height: 200px;
+    width: 200px;
+    display: flex;
+    justify-content: flex-start;
+}
+
+.box3{
+    background-color: red;
+    height: 200px;
+    width: 400px;
+}
+
+.box4{
+    background-color: blue;
+    height: 400px;
+    width: 200px;
+    margin-left: -200px;
+}
```

Pentru a acomoda schimbarea inrte Midle tile si Wide tile trebuie schimbat flex-direction pentru parinti + stergem margin-left: -200px;

```suggestion
.parent3 {
    display: flex;
    flex-direction: column;
    gap: 16px;
}

.parent2 {
    display: flex;
     flex-direction: row;
    gap: 16px;
}

.parent1 {
    display: flex;
    flex-direction: column;
    gap: 16px;
}

.box1 {
    background-color: turquoise;
    height: 200px;
    width: 200px;
}

.box2 {
    background-color: yellow;
    height: 200px;
    width: 200px;
    display: flex;
    justify-content: flex-start;
}

.box3 {
    background-color: red;
    height: 200px;
}

.box4 {
    background-color: blue;
    width: 200px;
}
```